docs/guides/admin/docs/configuration/multi.tenancy.md

```diff
@@ -75,7 +76,7 @@ do not support multitenancy.
 Note that if you are running Apache httpd with mod\_proxy in front of the Opencast installation, the port number will be
 -1 in both files.
 
-### Tenant-specific capture agent users
+### Step 2: Tenant-specific capture agent users
 
 If you use tenant-specific capture agent users, you should use the `prop.org.opencastproject.host.<server url>` options
```

Comment:
This section should be scratched. Since the `server` key no longer exists, _everyone_ needs to configure this for multi tenancy, regardless of whether they use tenant-specific capture agent users.